graylog2-server/src/main/java/org/graylog2/streams/StreamServiceImpl.java

```diff
@@ -340,7 +339,7 @@ public Set<String> indexSetIdsByIds(Collection<String> streamIds) {
                 .map(ObjectId::new)
                 .collect(Collectors.toSet());
         final DBObject query = QueryBuilder.start(StreamImpl.FIELD_ID).in(objectIds).get();
-        final DBObject onlyIndexSetIdField = DBProjection.include(FIELD_INDEX_SET_ID);
+        final DBObject onlyIndexSetIdField = new BasicDBObject(FIELD_INDEX_SET_ID, 1);
```

Comment:
I think we can use `com.mongodb.client.model.Projections.include()` here, no?

Reply:
We can't use it nicely because we must convert it afterward to be usable with the legacy API. But it improves the readability.